executor/executor.yml

```diff
@@ -218,4 +224,16 @@
 
 - debug:
     msg: "engine : {{ c_engine_yaml.stdout }}" 
- 
\ No newline at end of file
+
+
+- name: Get the jobCleanUpPolicy
+  shell: >
+    kubectl get chaosengine {{ c_engine }} -n {{ c_app_ns }} -o jsonpath='{.spec.jobCleanUpPolicy}'
+  args:
+    executable: /bin/bash
+  register: jobCleanUpPolicy
+
+- name: Delete job if the jobCleanUpPolicy status is delete
+  shell: >
+    kubectl delete job {{ job_name }} -n {{ c_app_ns }} 
+  when: jobCleanUpPolicy.stdout == "delete"
```

Comment:
I think this check will retain the completed jobs even if `jobCleanUpPolicy` is something other than `retain`. Will it be ok @ksatchit @chandankumar4

Reply:
There are a few more such scenarios where we will need stricter validation of the values provided in the schema. Raised issue #878 for this. @rahulchheda could you please enhance the executor README with note regarding the "supported" values, i.e., (example chaosEngine with this option specified)

Reply:
Thank you @rahulchheda @ksatchit .